### DISCUSSION TITLE
有个问题想问开发者团队，为什么当时考虑用React编写前端，而不是采用Vue编写

### DISCUSSION THREAD

**Discussion by jerry-rs** [clash-verge-rev/clash-verge-rev#2184] [Q&A] [answered]
正如题目的问题

**Comment by Dragon1573** [comment, marked as answer]
在某个平行世界，CVR 是用 Vue 开发的。

某一天 CVR 的 Discussions 里出现了一条问题：「有个问题想问开发者团队，为什么当时考虑用 Vue 编写前端，而不是采用 React 编写」。

-----

简而言之，没有为什么。

除非实际参与开发、改动项目核心源代码的 Collaborators / Members 愿意现身说法。
